ddtrace/bootstrap/sitecustomize.py

```diff
@@ -10,12 +10,20 @@
 
 from ddtrace.utils.formats import asbool, get_env
 
+logs_injection = asbool(get_env('logs', 'injection'))
+DD_LOG_FORMAT = '%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s'.format(
+    '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] ' if logs_injection else ''
+)
 
 debug = os.environ.get("DATADOG_TRACE_DEBUG")
+
+# Set here a default logging format for basicConfig since we would not be able
```

Comment:
This comment would be more clear if it called out the fact that once `basicConfig` is called, we cannot use it again to set a format string.

Reply:
How's the new version look?